lib/auth/auth.go

```diff
@@ -910,6 +931,57 @@ func (a *Server) updateVersionMetrics() {
 	}
 }
 
+var (
```

Comment:
I did that initially, but then for tests I needed a way to modify them to avoid creating 600+ remote clusters to hit dynamic limits: https://github.com/gravitational/teleport/pull/22455/files#diff-151919fe60e4dd4274a1c07382d3ae3c169dbeadedfd5eb10df0370421e50e29R121-R122